avm/res/purview/account/main.json

```
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.25.53.49325",
-      "templateHash": "11885849494373802730"
+      "version": "0.26.54.24096",
+      "templateHash": "6363425469984006749"
```

```suggestion
      "version": "0.25.53.49325",
      "templateHash": "11885849494373802730"
```

The readme change is good (be it curious) but I'd suggest to undo the JSON change as we'd not want to publish a new module version.

---

done